### ISSUE TITLE
Unmet peer dependency @babel/register@7.0.0

### ISSUE THREAD

**Issue by NN---** [mozilla/web-ext#1401]
### Is this a feature request or a bug?
Warning

Install web-ext package.

### What is the expected or desired behavior?
No such warnings

```
warning "web-ext > addons-linter > @babel/register@7.0.0" has unmet peer dependency "@babel/core@^7.0.0-0".
```

### Version information (for bug reports)
2.9.2

* **Your OS and version**: Windows 10
* Paste the output of these commands:

````
node --version && npm --version && web-ext --version
````
v11.1.0
6.4.1
2.9.2

**Comment by rpl** [comment]
This has been fixed in addons-linter version >= 1.4.2, this new versions of the linter are incompatible with nodejs 6, and so we will update the dependency in web-ext 3.0.0 (in which we are also going to deprecate support for nodejs 6).

**Comment by rpl** [comment]
This should now be fixed, the addons-linter dependency has been updated in #1503 and [web-ext 3.0.0 has been released on npm](https://github.com/mozilla/web-ext/releases/tag/3.0.0).
